amazonia-1/amazonia-1-tiler-example.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
<p>Can you put a comment explaining the importance of needing the center point in the mosaic call?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/asdi-examples/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>